components/Overview/utils.js

```diff
@@ -1,4 +1,23 @@
 import { ASSETS_SERVER_BASE_URL, RENDER_FRONTEND_BASE_URL } from '../../lib/constants'
 
 export const renderWidth = 1200
-export const getSmallImgSrc = teaser => `${ASSETS_SERVER_BASE_URL}/render?width=${renderWidth}&height=1&url=${encodeURIComponent(`${RENDER_FRONTEND_BASE_URL}/?extractId=${teaser.id}`)}&resize=160`
+export const getSmallImgSrc = teaser => `${ASSETS_SERVER_BASE_URL}/render?width=${renderWidth}&height=1&url=${encodeURIComponent(`${RENDER_FRONTEND_BASE_URL}/?extractId=${teaser.id}`)}&resize=200${teaser.contentHash ? `&permanentCacheKey=${encodeURIComponent(teaser.contentHash)}` : ''}`
```

Comment:
```suggestion
export const getSmallImgSrc = teaser => `${ASSETS_SERVER_BASE_URL}/render?viewport=${renderWidth}x1&url=${encodeURIComponent(`${RENDER_FRONTEND_BASE_URL}/?extractId=${teaser.id}`)}&resize=200${teaser.contentHash ? `&permanentCacheKey=${encodeURIComponent(teaser.contentHash)}` : ''}`
```
[width/height is deprecated](https://github.com/orbiting/backends/tree/master/packages/assets#endpoints)